app/scripts/lib/account-tracker.js

```diff
@@ -110,25 +111,147 @@ export default class AccountTracker {
         ) {
           this._updateAccounts();
         }
-      }, this.onboardingController.store.getState()),
+      }, this.preferencesController.store.getState()),
```

Comment:
Fixes potential bug with state comparison. Notice that `onboardingController` state was being used as the `prevState` value for `preferencesController` state changes

Reply:
LOL 👍